openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -296,6 +296,19 @@ jQuery(function () {
             });
     }
 
+    const coverForm = document.querySelector('.ol-cover-form--clipboard');
```

Comment:
Instead of a separate block here (which has some performance implications), since this is also int he `covers.js` file, follow the pattern above for eg `saveCoversElement`.